### ISSUE TITLE
Trigger on obstruction sensor

### ISSUE THREAD

**Issue by cypherstream** [hjdhjd/homebridge-ratgdo#6]
### Describe the feature request

I was looking to create an automation that if the obstruction sensor was tripped, maybe turn the garage light on for 5 minutes.

The options I see are for occupancy sensor / motion, but my Security+ 1.0 opener doesn’t have any specific motion sensors like some more advanced wall buttons have.

### Describe the proposed solution

When obstruction is triggered can we keep an eye on it so we can run automations based off of that?

its “off” 99.9% of the time.
![IMG_7481](https://github.com/hjdhjd/homebridge-ratgdo/assets/2373869/d98dc7bc-eb0d-404d-81f7-fa57199a1e18)

### Describe alternatives you have considered to the enhancement

I thought about researching other types of motion sensors, but why buy more stuff when I have perfectly good working garage door obstruction beam sensors.

**Comment by hjdhjd** [comment]
Obstruction state is already reported in HomeKit in HBR and you can already create automations off of it. Those capabilities are just not exposed in Apple’s Home app, but they are in many of the third party HomeKit controller apps out there (Eve, Home+, to name two).

TL;DR: this all already exists, as does the ability to create automations based on occupancy and motion detection. You can read the documentation for more, or feel free to ask questions on the ratgdo channel in the Homebridge Discord.
